cmd/main.go

```diff
@@ -92,12 +92,23 @@ func main() {
 		logger.Fatalw("Failed to load config file", zap.Error(err))
 	}
 
+	config, err := configManager.GetConfig()
+	if err != nil {
+		logger.Fatalw("Failed to get config", zap.Error(err))
+	}
+	var interval int
+	if config.AutoHeal {
+		interval = config.AutoHealTimeIntervalMs
+	} else {
+		interval = config.SyncTimeIntervalMs
+	}
```

Comment:
the `AutoHeal` parameter is a little confusing - if it's false, it seems like it should mean "don't auto-heal", but it actually means "use the default auto heal time interval". Perhaps we could just have the `AutoHealTimeIntervalMs` field, and if it's not set, it's essentially the "zero" value, i.e. 0, which means use the default?